### PR TITLE
Added docker-cakephp repo to Development Environment section

### PR DESCRIPTION
**Pull request by stefanvangastel**
Added docker-cakephp repo based on a email request.

https://github.com/stefanvangastel/docker-cakephp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,6 +388,7 @@ Additional lists you might find useful:
 - [PuPHPet](https://puphpet.com/) - Web interface for building a Vagrant + Puppet box.
 - [Puppet](https://puppetlabs.com/) - A server automation framework and application.
 - [Vagrant](https://www.vagrantup.com/) - A portable development environment utility.
+- [Docker](https://github.com/stefanvangastel/docker-cakephp) - CakePHP in a docker container environment.
```
Comment (dereuromark) [inline README.md]: Alph. order please.

**Comment by dereuromark** [comment]
👍

**Comment by dereuromark** [comment]
@stefanvangastel Can you please fix the order so we can merge?

**Comment by stefanvangastel** [comment]
@dereuromark Yeah, sorry. Will do asap.

**Comment by stefanvangastel** [comment]
Changed alph order

**Pull request merged**